src/main/java/org/squiddev/plethora/integration/chickens/MetaChickens.java

```diff
@@ -0,0 +1,89 @@
+package org.squiddev.plethora.integration.chickens;
+
+import com.google.common.collect.Maps;
+import com.setycz.chickens.ChickensMod;
+import com.setycz.chickens.config.ConfigHandler;
+import com.setycz.chickens.entity.EntityChickensChicken;
+import com.setycz.chickens.registry.ChickensRegistry;
+import com.setycz.chickens.registry.ChickensRegistryItem;
+import net.minecraft.entity.Entity;
+import net.minecraft.nbt.NBTTagCompound;
+import org.squiddev.plethora.api.Injects;
+import org.squiddev.plethora.api.converter.DynamicConverter;
+import org.squiddev.plethora.api.meta.IMetaProvider;
+import org.squiddev.plethora.api.meta.NamespacedMetaProvider;
+
+import javax.annotation.Nullable;
+import java.util.Map;
+
+@Injects(ChickensMod.MODID)
+public final class MetaChickens {
+	private MetaChickens() {
+	}
+
+	// A lambda expression would be more concise, but the line length is a bit much
```

Comment:
You can technically do something like this:
```java
public static final DynamicConverter<Entity, EntityChickensChicken> GET_CHICKEN_FROM_ENTITY = 
  entity -> entity instanceof EntityChickensChicken ? (EntityChickensChicken) entity : null;
```
Your call though.

Reply:
... :man_facepalming: Yeah, that'll work.